Doppler.BigQueryMicroservice/Controllers/BigQueryController.cs

```diff
@@ -13,19 +13,31 @@ namespace Doppler.BigQueryMicroservice.Controllers
     [ApiController]
     public class BigQueryController : ControllerBase
     {
-        private readonly IUnitOfWork unitOfWork;
+        private readonly IUnitOfWork _unitOfWork;
         public BigQueryController(IUnitOfWork unitOfWork)
         {
-            this.unitOfWork = unitOfWork;
+            this._unitOfWork = unitOfWork;
         }
 
         [HttpGet("/big-query/{accountName}/allow-emails")]
         public async Task<ActionResult> GetAllowEmails(string accountName)
         {
-            var data = await unitOfWork.UserAccessByUser.GetAllByUserIdAsync(accountName);
+            var data = await _unitOfWork.UserAccessByUser.GetAllByUserIdAsync(accountName);
             var result = new AllowEmails(data);
             return Ok(result);
+        }
 
+        [HttpPost("/big-query/{accountName}/allow-emails")]
```

Comment:
Acá estamos modificando el recurso que representa el listado de _"emails permitidos"_, por eso creo que el verbo correcto es `PUT`, la operación debería ser idempotente lo que confirma el uso de `PUT`.

```suggestion
        [HttpPut("/big-query/{accountName}/allowed-emails")]
```

Reply:
suggest acepted and applied